lesson-10/public/app.js

```diff
@@ -0,0 +1,62 @@
+$('#button').click(() => {
+  const name = $('#myName').val();
+  const comment = $('#myComment').val();
+  $.ajax({
+    type: 'POST',
+    url: '/submit',
+    contentType: 'application/json',
+    data: JSON.stringify({
+      name,
+      comment
+    }),
+    success: (data) => {
+      console.log('Success: ', data);
+    }
+  });
+});
+
+/* checking txt file every second */
+const getInboxFile = () => {
+ $.get('/notes.txt', (data) => {
+    const content = $('#chat');
+    content.html(data);
```

Comment:
if you are only gonna use content one time, why assigning it to a variable? It's like why having a function when you are only gonna use it once?